pkgs/by-name/ba/batik/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  jre,
+  rhino,
+  stripJavaArchivesHook,
+  makeWrapper,
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Use of `finalAttrs` is preferred over `rec` if this suggestion is accepted remember to close the newly opened parenthesis (see suggestion at the end of the pacckage)
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```